src/registrar/config/settings.py

```diff
@@ -300,7 +300,7 @@
 # Sets clients that allow access control to manage.get.gov
 # TODO: remove :8080 to see if we can have all localhost access
 CORS_ALLOWED_ORIGINS = ["http://localhost:8080", "https://beta.get.gov"]
-
+CORS_ALLOWED_ORIGIN_REGEXES = [r"https:\/\/[\w-]+\.sites\.pages\.cloud\.gov"]
```

Comment:
**question (really non-blocking)**: Did you find that the forward slashes needed to be escaped with backslashes? I didn't think they did, but I'd like to know if I was mistaken.

Reply:
Great question! We added it in case, but I just removed them in [this commit](https://github.com/cisagov/manage.get.gov/pull/1299/commits/802f8092a7c98f7b527390c905e9c9c415076d83) and seems like they're not necessary! TIL 🥳